campaigns.py

```diff
@@ -78,9 +78,9 @@ def deploy_banner(application):
     content = env['template_contents']
     put(StringIO.StringIO(content), remote_filename, use_sudo=True, mirror_local_mode=True)
     sudo('chown deploy:deploy %s' % remote_filename)
+    execute(app.restart, application)
```

Comment:
We only need to hard restart for `static`, because of the page caching issue. `frontend` can still be reloaded, which will mean fewer 500s.